sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/AppendBlobAsyncClient.java

```diff
@@ -199,32 +199,37 @@
      *
      * <p><strong>Code Samples</strong></p>
      *
-     * {@codesnippet com.azure.storage.blob.specialized.AppendBlobAsyncClient.appendBlockWithResponse#Flux-long-AppendBlobAccessConditions}
+     * {@codesnippet com.azure.storage.blob.specialized.AppendBlobAsyncClient.appendBlockWithResponse#Flux-long-byte-AppendBlobAccessConditions}
```

Comment:
@alzimmermsft is this the correct syntax to use for a byte array in these code snippet tags?

Reply:
It's the same as what was used in appendBlockFromURL, which already supported this.

Reply:
This is the correct syntax, generics and array types use the fix keyword only (byte[] -> byte, Flux<ByteBuffer> -> Flux).